docs/modules/ROOT/pages/advanced_usage/options.adoc

```diff
@@ -10,6 +10,7 @@ The other options are:
 [options="header"]
 |===
 | Option | Description
+| `--showsettings` or `-s` | Open the settings dialog while starting.
```

Comment:
Only a detail: we never use `or` when having a short and long option.
We would write:
|  `-s` +
`--showsettings` | Open the settings dialog while starting

See https://doc.owncloud.com/server/10.3/admin_manual/configuration/server/occ_command.html#background-jobs-selector
Executing a background job

Reply:
I know you will not like it, but you have to make a new line after the plus symbol, exactly as written. This is due to the way how ascidoc works. The `+` sign at the end of a line shows, that the next line is part of that block but a new line (like you have in MS Word shift-return making a new line in the same paragraph while a return makes a new line with a new paragraph). Having that, the table renders correctly.

Reply:
Fixed, thanks for explaining me. (I thought the + was meant as a literal +)

Reply:
Approved from the docs pov 😄 
Marking as resolved